download-qemu-static.sh

```diff
@@ -1,33 +1,43 @@
 #!/bin/bash -ex
 
-set -xe
+set -eux
+
 if [ "$(uname -m)" == "x86_64" ]; then
     docker run --rm --privileged multiarch/qemu-user-static:register --reset
```

Comment:
N.B.: We can leave this as is since the `binfmt_misc` registration will be the same for any version.